src/javascript/app/pages/cashier/cashier.js

```diff
@@ -66,9 +66,12 @@ const Cashier = (() => {
         const top_up_id = '#VRT_topup_link';
         const $a        = $(top_up_id);
         if (!$a) return;
-        const new_el    = { class: 'toggle button', html: $a.html(), id: $a.attr('id') };
-        href            = href || Url.urlFor('/cashier/top_up_virtualws');
-        new_el.href     = href;
+        let new_el          = { class: 'toggle button', html: $a.html(), id: $a.attr('id') };
+        href                = href || Url.urlFor('/cashier/top_up_virtualws');
+        new_el.href         = href;
+        if (Client.get('balance') === 10000) {
+            new_el = { id: $a.attr('id'), class: 'toggle button button-disabled', html: $a.html() };
```

Comment:
why not just update the class param in new_el object and leave the rest since they are unchanged?